Fix build with opaque RSA in LibreSSL 3.5.

XXX: maybe try calling OPENSSL_cleanup() in crypto_lib_exit() for LibreSSL.
May prevent leaks?

Index: packages/ssl/crypto4pl.c
--- packages/ssl/crypto4pl.c.orig
+++ packages/ssl/crypto4pl.c
@@ -892,7 +892,7 @@ recover_rsa(term_t t, RSAKEY** keyp)
   RSAKEY *key = RSA_new();
 #endif
 
-#if OPENSSL_VERSION_NUMBER < 0x10100000L || defined(LIBRESSL_VERSION_NUMBER)
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
   if ( get_bn_arg(1, t, &key->n) &&
        get_bn_arg(2, t, &key->e) &&
        get_bn_arg(3, t, &key->d) &&
